snmp/datadog_checks/snmp/data/default_profiles/fortinet-fortigate.yaml

```diff
@@ -190,7 +190,7 @@ metrics:
       OID: 1.3.6.1.4.1.12356.101.3.2.1
       name: fgVdTable
     symbols:
-      - name: fgVirtualDomain
+      - name: fgVd
```

Comment:
I think we can leave fgVirtualDomain (real name fgVd) since it's more readable.
I don't like exceptions, but this case seems probably fine...

Reply:
Agree